heartbeat/aws-vpc-route53.in

```diff
@@ -213,12 +213,31 @@ r53_monitor() {
 	if [ "$__OCF_ACTION" = "start" ] || ocf_is_probe ; then
 		#
 		cmd="aws $AWS_PROFILE_OPT route53 list-resource-record-sets --hosted-zone-id $OCF_RESKEY_hostedzoneid --query ResourceRecordSets[?Name=='$OCF_RESKEY_fullname']"
-		ocf_log debug "Route53 Agent Starting or probing - executing monitoring API call: $cmd"
-		ARECORD="$($cmd | grep RESOURCERECORDS | awk '{ print $2 }')"
+		ocf_log info "Route53 Agent Starting or probing - executing monitoring API call: $cmd"
```

Comment:
I think we should keep these as debug to avoid filling the logfiles with too much info by default.

Reply:
Hello @oalbrigt,

I rather keep this one as "info". Amount of log is minimum as it is a single line only during start or probe.

Thanks!

Reply:
Makes sense.